Bio/Sequencing/Applications/_bwa.py

```diff
@@ -380,7 +380,118 @@ def __init__(self, cmd="bwa", **kwargs):
                     "Minimum number of seeds supporting the resultant alignment to skip reverse alignment. [5]",
                     checker_function=lambda x: isinstance(x, int),
                     equate=False),
-            ]
+        ]
+        AbstractCommandline.__init__(self, cmd, **kwargs)
+
+
+class BwaMemCommandline(AbstractCommandline):
+    """Command line wrapper for Burrows Wheeler Aligner (BWA) mem.
+
+    Run a BWA-MEM alignment, with single- or paired-end reads, equivalent to::
+
+        $ bwa mem [...] <in.db.fasta> <in1.fq> <in2.fq> > <out.sam>
+
+    See http://bio-bwa.sourceforge.net/bwa.shtml for details.
+
+    Examples
+    --------
+    >>> from Bio.Sequencing.Applications import BwaMemCommandline
```

Comment:
You need a blank line before the doctest for it to be formatted correctly as RST in our API documentation generated from the source code.

Reply:
All classes in the file lack the blank lines you mentioned; will add them.